source/graphql/schema/resolver.d

```diff
@@ -159,8 +160,8 @@ class __Schema {
 
 void setDefaultSchemaResolver(T, Con)(GraphQLD!(T,Con) graphql) {
 
-	Json typeResolverImpl(Type)(ref const(StringTypeStrip) stripType,
-			Json parent)
+	static Json typeResolverImpl(Type)(ref const(StringTypeStrip) stripType,
```

Comment:
same comment as above, static limits how many GraphQLD instances you
can have

Reply:
`static` in this instance makes it not an inner function with access to the outer context. It doesn't compile if I don't do this, because there are too many context pointers and the compiler can't handle it.